railties/lib/rails/generators/rails/plugin/templates/Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
```

Comment:
Should not this be double quotes too?

Reply:
yes, I think this should be double quotes too

Reply:
but I deleted the fork so I can't change it now. Should I open a new pull request if you agree about this change?